app/views/shared/bulk_updates/display/_science_keywords.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if bulk_update['update-type'] == 'FIND_AND_REMOVE' || bulk_update['update-type'] == 'FIND_AND_REPLACE' %>
+  <!-- Find Value -->
+  <%= display_value_title(bulk_update['update-type'], 'find') %>
+
+  <ul class="arrow-tag-group-list">
+    <% display_science_keyword(bulk_update['find-value']).each do |field, value| %>
+      <li itemprop="keyword" class="arrow-tag-group-item"><%= value %></li>
+    <% end %>
+  </ul>
+<% end %>
+
+<% if bulk_update['update-type'] == 'ADD_TO_EXISTING' || bulk_update['update-type'] == 'CLEAR_ALL_AND_REPLACE' %>
```

Comment:
the wiki says "If NOT type FIND_AND_REMOVE, New value required", (and FIND_AND_REPLACE should have the new aka update value), so that might be a better test here